content/doc/developer/plugin-governance/adopt-a-plugin.adoc

```diff
@@ -20,23 +20,44 @@ This page addresses cases when there is no active maintainer of a plugin:
 
 == Requesting commit access
 
-Email the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list] and request to be made a maintainer 
-(which usually means commit access to the plugin repository and being made default assignee for newly reported issues in JIRA).
-Please provide the following things in the request:
+=== Plugins marked for adoption
+
+The procedure is two-fold: 
+
+==== announce your adoption intention by mail 
+
+Send the notification email to the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list].
+Be sure to provide the following information: 
 
 * Link to a plugin you want to adopt
+* The status of the plugin ("for adoption" or "abandonned")
 * Link(s) to pull requests you want to deliver, if applicable
 * Your GitHub username/id (e.g. oleg-nenashev for https://github.com/oleg-nenashev/)
 * Your Jenkins infrastructure account id. link:https://accounts.jenkins.io/[Create your account] if you don't have one.
+* The link to the "Repository Permission Updater" PR described below
+
+==== submit a PR on the "Repository Permission Updater" (RPU) repository
+
+File a pull request against link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permission updater] to be able to deploy snapshots and releases for your plugin.
+You need to update the plugin's definition yaml file located in the `permission` directory.
+Add your Jenkins infrastructure account under the `developer` section.
+Refer to the link:https://github.com/jenkins-infra/repository-permissions-updater/blob/master/README.md[Readme] for detailed guidance.
+
+It is a good practice to ping the current registered developers on the PR (just in case they miss the notification pn the Jenkins Dev List).
```

Comment:
```suggestion
It is a good practice to ping the current registered developers on the PR (just in case they miss the notification on the Jenkins Dev List).
```